python/test/hail/test_ir.py

```diff
@@ -129,11 +129,11 @@ def table_irs(self):
             ir.MatrixRowsTable(matrix_read),
             ir.TableParallelize(ir.Literal(hl.tarray(hl.tstruct(a=hl.tint32)), [{'a':None}, {'a':5}, {'a':-3}]), None),
             ir.TableMapRows(
-                table_read,
+                ir.TableUnkey(table_read),
```

Comment:
Why is the Unkey necessary?it looks like we don't modify the keys

Reply:
Because `table_read` has keys, and the `TableMapRows` is giving the result no keys. The two fixes are to unkey before (what I did), or to put the existing keys of `table_read` in the `TableMapRows` node. I didn't see any reason to prefer one vs the other; I went with this to avoid having to figure out the keys of `table_read`, and because it seems slightly less brittle in case the table being read ever changed keys.